glide-core/src/client/value_conversion.rs

```diff
@@ -607,6 +608,50 @@ pub(crate) fn convert_to_expected_type(
             }
             _ => Err((ErrorKind::TypeError, "Response couldn't be converted").into()),
         },
+        // Used by XAUTOCLAIM. The command returns a list of length 2 if the Redis version is less than 7.0.0 or a list
+        // of length 3 otherwise. It has the following response format:
+        /* Redis version < 7.0.0 example:
+        1) "0-0"
+        2) 1) 1) "1-0"
+              2) 1) "field1"
+                 2) "value1"
+           2) 1) "1-1"
+              2) 1) "field2"
+                 2) "value2"
+
+        Redis version >= 7.0.0 example:
+        1) "0-0"
+        2) 1) 1) "1-0"
+              2) 1) "field1"
+                 2) "value1"
+           2) 1) "1-1"
+              2) 1) "field2"
+                 2) "value2"
+        3) 1) "1-1"
```

Comment:
Looks like a truncated response. What it the meaning of this piece of response?

Reply:
The third element in the array response here is an array containing the stream entry IDs that were in the Pending Entry List but no longer in the stream itself